Readme.md

```diff
@@ -37,3 +49,15 @@ fn main() {
 Version 2 would add `list`, `purge_all`, and `restore_all` that would allow for listing, permanently removing or restoring trashed items.
 Development for Version 2 is currently suspended as I couldn't manage to get these features to work on Mac. Contribution would be very welcome.
 An imperfect alternative would be to release those features for Linux and Windows both of which at this point have a more or less complete implementation of these features on the `v2-dev` branch. The windows implementation depends on a custom fork of winapi, because the PR adding the required features cannot be merged before winapi 0.4. I would like to merge the branch adding `list`, `purge_all`, and `restore_all` only after the winapi PR is is merged and published.
+
+## Apps using trash
+
+- [emulsion](https://github.com/ArturKovacs/emulsion)
+- [nushell](https://github.com/nushell/nushell)
```

Comment:
I think this is redundant as this can be found at crates.io at the dependent crates tab. As such this just adds clutter.